src/map/skill.c

```diff
@@ -14942,7 +14943,8 @@ static int skill_unit_effect(struct block_list *bl, va_list ap)
 
 	nullpo_ret(group);
 
-	dissonance = skill->dance_switch(su, 0);
+	if (!(flag & 8))
```

Comment:
`if ((flag & 8) == 0)` please. Technically not working with a boolean here.

Reply:
Done.